internal/accounts/create.go

```diff
@@ -213,32 +228,12 @@ func createInteractive(state *flowkit.State, loader flowkit.ReaderWriter) (*flow
 
 		address = account.Address
 	} else {
-		var link string
-		switch selectedNetwork {
-		case config.DefaultTestnetNetwork():
-			outputList(log, []string{
-				"Please complete the following steps in a web browser",
-				"Complete the captcha challenge.",
-				"Click the 'Create Account' button.",
-				"Return to this window.",
-			}, true)
-			link = util.TestnetFaucetURL(key.PublicKey().String(), crypto.ECDSA_P256)
-
-		case config.DefaultMainnetNetwork():
-			outputList(log, []string{
-				"Please complete the following steps in a web browser",
-				"Click on 'Submit' button.",
-				"Connect existing Blocto account or create new.",
-				"Click on confirm and approve transaction.",
-			}, true)
-			link = util.MainnetFlowPortURL(key.PublicKey().String())
-		}
-
-		output.ConfirmOpenBrowser()
+		log.StartProgress("Please wait while we create your new account....\n")
 
-		log.StartProgress("Waiting for your account to be created, please finish all the steps in the browser...\n")
-		_ = util.OpenBrowserWindow(link)
-		log.Info(output.Italic(fmt.Sprintf("You can also navigate to the link manually: %s\n", link)))
+		err = createNewAccount(selectedNetwork.Name, key.PublicKey().String())
+		if err != nil {
+			return nil, err
+		}
 
 		addr, err := getAccountCreatedAddressWithPubKey(service, key.PublicKey(), startHeight)
```

Comment:
the API returns transaction ID as response, but I think we can just keep the current implementation of scanning through the events and match it with the public key

Reply:
We should track that transaction ID, it's more optimal and reliable.